support-frontend/assets/helpers/page/page.ts

```diff
@@ -15,6 +15,7 @@ import { getReferrerAcquisitionData } from 'helpers/tracking/acquisitions';
 import { getSettings } from '../globalsAndSwitches/globals';
 
 function setUpTrackingAndConsents(participations: Participations): void {
+	console.log({ participations });
```

Comment:
moving this log to here because otherwise it would not include the landing page tests